test/apiMock.js

```diff
@@ -97,7 +96,7 @@ export function initApiMockServer () {
 export function initApiMock () {
   initApiMockServer()
 
-  Vue.use(Vuex)
+  // Vue.use(Vuex)
```

Comment:
This should be deleted I guess.

Reply:
Done in 613270bba5d7c0ef64ea36d0373a6fd3af8c982c